### DISCUSSION TITLE
How run windows Client As Service ???

### DISCUSSION THREAD

**Discussion by raminmalekghasemi** [apernet/hysteria#550] [General]
Hi Dear Friends I used this

sc.exe create proxy-new binPath= "c:/proxy/hysteria-tun-windows-6.0-amd64.exe"

To start service get error Doonot Start Or Control On exe file
Hysteria server information

ubuntu 16.0.4
VPS information

1GRAM
Server config

Server Config IS ok
Server logs

Server No problem..
Hysteria client information

000000
Client config

{
"server": "CN.NaME:2869",
"obfs": "My.PASSWOD",
"up_mbps": 1000,
"down_mbps": 1000,
"socks5": {
"listen": "127.0.0.1:1080"
},
"http": {
"listen": "127.0.0.1:1081"
}
}
Client environment (operating system)

windows 7
Client logs

1053

**Comment by haruue** [comment]
The actual error should be (in the services.msc)
> Error 1053: The service did not respond to the start or control request in a timely fashion

And I don't think hysteria would work in this way.

> Note that it will not work for just any executable: the executable must be a Windows Service (i.e. implement ServiceMain).

https://stackoverflow.com/a/3582179

However, there are some third-party tools that can be used to register hysteria as a system service, you 
can find them in the other answers of the above StackOverflow question.
